velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -653,16 +658,29 @@ std::vector<core::TypedExprPtr> ExpressionFuzzer::generateArgs(
             0, FLAGS_max_num_varargs)(rng_);
   inputExpressions.reserve(input.args.size() + numVarArgs);
 
-  for (const auto& arg : input.args) {
-    inputExpressions.emplace_back(generateArg(arg));
+  for (auto i = 0; i < input.args.size(); ++i) {
+    inputExpressions.emplace_back(
+        generateArgs(input.args.at(i), input.constantArgs.at(i)));
   }
+
   // Append varargs to the argument list.
   for (int i = 0; i < numVarArgs; i++) {
-    inputExpressions.emplace_back(generateArg(input.args.back()));
+    inputExpressions.emplace_back(
+        generateArgs(input.args.back(), input.constantArgs.back()));
   }
   return inputExpressions;
 }
 
+core::TypedExprPtr ExpressionFuzzer::generateArgs(
```

Comment:
generateArgs -> generateArg, since it generates a single argument

Reply:
Fixed